assets/resourceSnippets/Automanage Best Practices DevTest Assignment.snippet.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Could you please merge this PR?  Thx 
https://github.com/AndrewCS149/vscode-azurearmtools/pull/1